src/components/PsaBanner/index.tsx

```diff
@@ -15,23 +16,32 @@ const redirectToNewApp = (): void => {
   window.location.replace(NEW_URL + path)
 }
 
-const BANNERS: Record<string, ReactElement | string> = {
-  '*': (
+const BANNERS: Record<string, ReactElement | string> = {}
+
+const NO_REDIRECT_PARAM = 'no-redirect'
+
+const WebCoreBanner = (): ReactElement | null => {
+  const { search } = useLocation()
+  const shouldRedirect = !new URLSearchParams(search).get(NO_REDIRECT_PARAM)
+
+  return (
     <>
       ⚠️ Safe&apos;s new official URL is <a href={NEW_URL}>app.safe.global</a>.<br />
       Please update your bookmarks.{' '}
-      <Countdown seconds={10} onEnd={redirectToNewApp}>
-        {(count) => <>Redirecting in {count} seconds...</>}
-      </Countdown>
+      {shouldRedirect && (
+        <Countdown seconds={10} onEnd={redirectToNewApp}>
+          {(count) => <>Redirecting in {count} seconds...</>}
+        </Countdown>
+      )}
     </>
-  ),
+  )
 }
 
 const WARNING_BANNER = 'WARNING_BANNER'
 
 const PsaBanner = (): ReactElement | null => {
   const chainId = useSelector(currentChainId)
-  const banner = BANNERS[chainId] || BANNERS['*']
+  const banner = BANNERS[chainId] || BANNERS['*'] || <WebCoreBanner />
```

Comment:
Why change this? Remove the * then.

Reply:
The banner need access the from `useLocation.search`. I've removed `BANNERS['*']`.